scripts/check-usage-of-log-warn.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# shellcheck disable=SC2038
+OUTPUT=$(find . -wholename "*/migrations/*.rs" | xargs grep "log::warn" || true)
```

Comment:
How useful it is to check if there is explicit log::warn usage in the file? Not sure I understand what is it trying to prevent. After macro expansion, both cases will have log::warn but the latter will also panic. Logically speaking, both contain the logs. If you are trying to prevent panicking when not  `"try-runtime"`, there should be better ways to stop compilation programmatically. Not saying the current impl is wrong.

Reply:
This script will trigger a CI failure if we're using `log::warn!` inside migrations directly instead of using the `warn_or_panic` macro. 

In case we make a mistake and don't use `warn_or_panic` we risk that a migration fails during testing but we don't get alerted about that!